app/scripts/utils/__tests__/path-test.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env mocha */
```

Comment:
+1 for top level tests folder.  That's how most projects, and most of our projects do it I believe.